### PR TITLE
 Added 2 checkboxes in the anchor popover.

### PR DESCRIPTION
**Pull request by alekshs**
We can now choose whether the link will open in new tab and toggle a nofollow rel for SEO purposes

### CODE REVIEW

**Comment by MrAvantiC** [comment]
Hi, 
is there a specific reason why this never got merged?
I'm asking because - even though react-rte integrates quite nice with react-apps in our tests - this missing feature is one of two (besides missing text-colors) that current ykeeps us from using this library in our application. :/

File: src/ui/InputPopover.css [hunk 1]
```diff
@@ -58,3 +58,19 @@
...
+  padding: 8px 2px;
+}
+
+.targetBlank input, .noFollow input {
+  margin-right: 8px;
+}
+
+.targetBlank input, .noFollow label {
+  cursor: pointer;
+}
```
Comment (sstur) [inline src/ui/InputPopover.css:76]:
```suggestion
}

```

File: src/ui/InputPopover.js [hunk 2]
```diff
@@ -64,6 +76,30 @@ export default class InputPopover extends Component {
             />
           </ButtonGroup>
         </div>
+        {(props.target || props.target === null) &&
+          <div className={styles.targetBlank}>
+            <label>
+              <input
+                type="checkbox"
+                checked={this.state.target === '_blank'}
+                onChange={this._onNewTabPress}
```
Comment (sstur) [inline src/ui/InputPopover.js:85]:
```suggestion
                onChange={this._onTargetBlankPress}
```

File: src/ui/InputPopover.js [hunk 3]
```diff
@@ -64,6 +76,30 @@ export default class InputPopover extends Component {
...
+              />
+              <span>Open link in a new tab</span>
```
Comment (sstur) [inline src/ui/InputPopover.js:87]:
```suggestion
              <span>Open link in new tab</span>
```

File: src/ui/InputPopover.js [hunk 4]
```diff
@@ -72,6 +108,18 @@ export default class InputPopover extends Component {
     this._inputRef = inputElement;
   }
 
+  _onNewTabPress(event: Object) {
```
Comment (sstur) [inline src/ui/InputPopover.js:111]:
```suggestion
	  _onTargetBlankPress(event: Object) {
```

File: src/ui/PopoverIconButton.js [hunk 5]
```diff
@@ -37,6 +39,8 @@ export default class PopoverIconButton extends Component {
     return (
       <InputPopover
         defaultValue={this.props.defaultValue}
+        target={this.props.target}
+        rel={this.props.rel}
```
Comment (sstur) [inline src/ui/PopoverIconButton.js:43]: Same as above, these props are too specific to implement on `<InputPopover>`

File: src/lib/EditorToolbar.js [hunk 6]
```diff
@@ -208,6 +210,8 @@ export default class EditorToolbar extends Component {
           onTogglePopover={this._toggleShowLinkInput}
           defaultValue={defaultValue}
           onSubmit={this._setLink}
+          target={target}
+          rel={rel}
```
Comment (sstur) [inline src/lib/EditorToolbar.js:214]: Hmm, these props seem too specific to implement on `< PopoverIconButton >`

**Pull request merged**